sdk/template/azure-template-stress/README.md

```diff
@@ -0,0 +1,228 @@
+# Stress tests for Azure <Name> client library for Java
+
+This package contains template project for stress tests and recommendations on how to create them for your library.
+
+## Getting started
+
+Check out [Azure SDK Stress Test Wiki][azure_sdk_stress_test] for general information about stress tests.
+
+### Prerequisites
+
+- [Java Development Kit (JDK)][jdk_link], version 8 or later.
+- [Maven][maven]
+- [Docker][docker]
+- [Kubectl][kubectl]
+- [Helm][helm]
+- [Azure CLI][azure_cli]
+- [Powershell 7.0+][powershell]
+
+### Deploy Stress Test
+
+cd into `azure-sdk-for-java` root folder and run command to deploy the package to cluster:
+
+```shell
+.\eng\common\scripts\stress-testing\deploy-stress-tests.ps1 -SearchDirectory .\sdk\<you service directory>
+```
+
+### Check Status
+
+Only the most frequently used commands are listed below. See [Deploying A Stress Test][deploy_stress_test] for more details.
+
+List deployed packages:
+
+```shell
+helm list -n <stress test namespace>
+```
+
+the namespace usually matches your username.
+
+Get stress test pods and status:
+
+```shell
+kubectl get pods -n <stress test namespace>
+```
+
+Get stress test pod logs:
+
+```shell
+kubectl logs -n <stress test namespace> <stress test pod name>
+# Note that we may define multiple containers (for example, `fault-injector` and `main`)
+kubectl logs -n <stress test namespace> <stress test pod name> -c <container name>
+```
+
+If stress test pod is in `Error` status, check logs from containers:
+
+```shell
+kubectl logs -n <stress test namespace> <stress test pod name>
+```
+
+You may also get logs for specific containers:
```

Comment:
This doesn't get logs, it's more like "show me a readable version of the metadata of the pod". 

So maybe this more of a "if you think the error is Kubernetes related describe will tell you more?"

Reply:
good catch, thanks!